src/NzbDrone.Mono/ProcMountProvider.cs

```diff
@@ -94,7 +98,7 @@ private IMount ParseLine(string line)
             }
 
             var name = split[0];
-            var mount = split[1];
+            var mount = GetMount(split[1]);
```

Comment:
Why not pipe the `split.Select(ExpandEscapes).ToArray()`? Strictly speaking these escapes could be in any field.